tests/proc/TEST_ComputeACF.m

```diff
@@ -0,0 +1,58 @@
+% setup
+rng(0); % for reproducibility
+
+% tuning variable: linear increase
+tuning_dt = 1/30; 
+tuning_var_tvec = 0:tuning_dt:100;
+tuning_var_data = linspace(80, 400, length(tuning_var_tvec));
+
+% spike train (logical)
+spike_dt = 0.001;
+spike_rate = 0.5;
+spike_tvec = 0:spike_dt:100;
+spike_data_binary = rand(size(spike_tvec)) > spike_rate;
+
+% spike train (continuous)
+spike_data_cont = rand(size(spike_tvec));
+
+cfg.bins = 0:10:640;
+cfg.interp = 'nearest';
+
+%% test that outputs have the expected size
+cfg_acf = [];
+cfg_acf.binsize = mean(diff(cfg.bins));
+
+[acf,tvec] = ComputeACF(cfg_acf,spike_data_binary); 
+assert(mean(diff(tvec)) == cfg_acf.binsize); % cfg.binsive should be the same as the tvec dt
+assert(tvec(ceil(end/2))==0) % tvec centered on 0
+assert(acf(ceil(end/2))==0) % acf centered on 0
+
+
+cfg_acf = [];
+cfg_acf.binsize = mean(diff(cfg.bins));
```

Comment:
this was previously defined in line 23 -- can you avoid repetition?

Reply:
Will continue to clear cfg_cfg but will use constants for fields.